### ISSUE TITLE
terraform-provider-kubernetes release schedule

### ISSUE THREAD

**Issue by vmasutin** [terraform-providers/terraform-provider-kubernetes#392]
Hello

Is there any release schedule for terraform-provider-kubernetes ? Does it depends on the number of features or/and bugfixes. It would be great to have some notes about this in the README.md.

Regards,
Valera

**Comment by alexsomesan** [comment]
Hi, 

Right now there is no formal release schedule. This is mostly because the influx of changes is not very consistent, so the default choice is to wait until a certain amount of changes accumulate.

Of course, more often releases are not hard to implement if there is enough demand for that.

Are you blocked on any unreleased changes so far?

**Comment by vmasutin** [comment]
Hi

I am particularly interested in this commit https://github.com/terraform-providers/terraform-provider-kubernetes/commit/cfaa1f02e6cecc2351bd56fd4632f68fcb85815c
It was merged into master 8 days ago and I am trying to get an idea on when to expect next release.
This small change has somewhat big implications :-)

**Comment by pasali** [comment]
Hi, i am waiting for this (ee47b93c035892aad33ddfc7464caad3889d3654) as well.

**Comment by alexsomesan** [comment]
I am aiming for a release next week. I am on leave until Monday, and will do a release as soon as possible next week.

**Comment by tete17** [comment]
I am also stuck for 5b77db0066df9b892e44da22a9722d577360c85c

**Comment by tdmalone** [comment]
The above-mentioned commits have since all been released in 1.6.0 (https://github.com/terraform-providers/terraform-provider-kubernetes/blob/master/CHANGELOG.md#160-april-17-2019)

**Comment by hashibot** [comment]
This issue has been open 180 days with no activity. If this issue is reproducible with the latest version of the provider and with Terraform 0.12, please comment. Otherwise this issue will be closed in 30 days.
